react/src/components/atoms/forms/Input/validate.js

```diff
@@ -0,0 +1,16 @@
+export const validNumber = (num, min, max, erroMsg) => {
+  if (!Number.isNaN(Number(min))) {
+    // Make sure the new value isn't less than the min value.
+    if (num < Number(min)) {
+      errorMsg = `Please enter a valid greater than or equal to ${min}.`;
```

Comment:
is this supposed to say?
```suggestion
      errorMsg = `Please enter a valid number greater than or equal to ${min}.`;
```